test/fixtures/config

```diff
@@ -0,0 +1,4 @@
+jobs:
+  : 1
```

Comment:
Oops... (while `jobs` isn't a valid config setting anyway,) this isn't supposed to look like that.

I think this:
https://github.com/gel-rb/gel/blob/6edbbe0a8e4957ec868e0b6aa37aceede6bd044e/lib/gel/config.rb#L20

needs to be followed by something like:
```ruby
group, key = nil, group if key.nil?
```

On the other hand, maybe we should change the file format (or group/key format?) to something different. The current behaviour looks not-great when `key` is a DNS name, as #73's example shows for `packages.shopify.io`. But that's a thought for another day.

Reply:
With `bundle config local.GEM_NAME /path/to/local/git/repository` and `bundle config jobs 5` we get the output:

```
.bundle % cat config
---
BUNDLE_GEM__TEST: "minitest"
BUNDLE_JOBS: "7"
BUNDLE_LOCAL__GEM_NAME: "/path/to/local/git/repository"
```

To be honest bundlers output is not great either - WDYT? I'm going to give this a little more thought.